### PR TITLE
delete binary

### PR DESCRIPTION
**Pull request by f1yegor**
delete binary api

### CODE REVIEW

**Comment by noorul** [comment]
@f1yegor This has reduced the test coverage. See https://codecov.io/gh/spark-jobserver/spark-jobserver/compare/6d16f6a01473cdf971c01d99be12ae0774b0a228...ec174fd6aead14ff92ae5a4feba65cfd2ef471f5

**Comment by f1yegor** [comment]
https://travis-ci.org/f1yegor/spark-jobserver

**Comment by noorul** [comment]
@f1yegor Tests are flaky.

**Comment by f1yegor** [comment]
Couldn't find what tests are failing.

**Comment by noorul** [comment]
@f1yegor Did you look at https://s3.amazonaws.com/archive.travis-ci.org/jobs/167517328/log.txt

**Comment by noorul** [comment]
@f1yegor Can you re-base this with master and push?

**Comment by f1yegor** [comment]
@noorul seems legit

File: job-server/src/main/scala/spark/jobserver/WebApi.scala [hunk 1]
```diff
@@ -255,6 +255,19 @@ class WebApi(system: ActorSystem,
...
           }
         }
+      } ~
+      // DELETE /binaries/<appName>
+      delete {
+        path(Segment) { appName =>
+          val future = binaryManager ? DeleteBinary(appName)
+          respondWithMediaType(MediaTypes.`application/json`) { ctx =>
+            future.map {
+              case BinaryDeleted => ctx.complete(StatusCodes.OK)
```
Comment (maxosprojects) [inline job-server/src/main/scala/spark/jobserver/WebApi.scala]: This would return "OK", which is not a valid json

Comment (f1yegor) [inline job-server/src/main/scala/spark/jobserver/WebApi.scala]: this way api works now. can be improved overall project in separate PR.

**Comment by noorul** [comment]
@f1yegor Shouldn't we document this new API in README.md?

**Comment by noorul** [comment]
@f1yegor Thank you!

**Pull request merged**